### ISSUE TITLE
tqdm behaviour

### ISSUE THREAD

**Issue by Bankimchandrayadav** [microsoft/vscode-jupyter#4124]
## Environment data

-   VS Code version: 1.52-insider
-   Jupyter Extension version (available under the Extensions sidebar): 2020.11.3
-   Python Extension version (available under the Extensions sidebar): 2020.11.2
-   OS (Windows | Mac | Linux distro) and version: Windows
-   Python and/or Anaconda version: Python 3.8.5
-   Type of virtual environment used : conda
-   Jupyter server running: Local 

## Expected behaviour

A full progress bar

## Actual behaviour

message about hbox description appearing

## Steps to reproduce:
Sample code to reproduce that behaviour: 
`from tqdm.notebook import notebook as td`
`for i in td(range(0, 100)):` 
`    c=1`
![Screenshot 2020-12-03 180044](https://user-images.githubusercontent.com/13675730/101114642-ed5e8e00-3607-11eb-8b71-a710c2593bd3.png)

[**NOTE**: Self-contained, minimal reproducing code samples are **extremely** helpful and will expedite addressing your issue]

1. XXX

## Logs

<details>

<summary>Output for <code>Jupyter</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Jupyter</code>)
</summary>

<p>

```
User belongs to experiment group 'jupyterTest'
User belongs to experiment group 'NativeNotebookEditor'
> C:\ProgramData\Miniconda3\python.exe -c "import os;print(os.path.realpath("C:\\Users\\bcyadav\\AppData\\Roaming\\jupyter\\kernels"))"
> C:\ProgramData\Miniconda3\python.exe -c "import os;print(os.path.realpath("C:\\Users\\bcyadav\\AppData\\Roaming\\jupyter\\kernels"))"
> ~\AppData\Local\ESRI\conda\envs\arcgispro-py3-clone\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py -c "import jupyter"
> ~\AppData\Local\ESRI\conda\envs\arcgispro-py3-clone\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py -c "import notebook"
> ~\AppData\Local\ESRI\conda\envs\arcgispro-py3-clone\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py jupyter kernelspec --version
> C:\ProgramData\Miniconda3\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py -c "import ipykernel"
> ~\AppData\Local\ESRI\conda\envs\arcgispro-py3-clone\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\vscode_datascience_helpers\getJupyterKernelspecVersion.py
> ~\AppData\Local\ESRI\conda\envs\arcgispro-py3-clone\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.jupyter_daemon -v
> ~\AppData\Local\ESRI\conda\envs\arcgispro-py3-clone\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.jupyter_daemon -v
> ~\AppData\Local\ESRI\conda\envs\arcgispro-py3-clone\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.jupyter_daemon -v
> C:\ProgramData\Miniconda3\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.kernel_launcher_daemon -v
> C:\ProgramData\Miniconda3\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py -c "import ipykernel"
> C:\ProgramData\Miniconda3\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py -c "import ipykernel"
Started kernel Python 3
> C:\ProgramData\Miniconda3\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.kernel_launcher_daemon -v
> C:\ProgramData\Miniconda3\python.exe c:\Users\bcyadav\.vscode-insiders\extensions\ms-toolsai.jupyter-2020.11.399280825\pythonFiles\pyvsc-run-isolated.py vscode_datascience_helpers.daemon --daemon-module=vscode_datascience_helpers.kernel_launcher_daemon -v
```

</p>
</details>

**Comment by joyceerhl** [comment]
I believe this was fixed in ee74e6771fca872eff3c88521d6e9379f2a758ad and the fix will become available in our December release. (@donjayamanne please let me know if I'm mistaken.) With dev bits and the renderer extension:

![image](https://user-images.githubusercontent.com/30305945/101315008-9a0a5c80-380e-11eb-83a7-1c9aaf4e2c74.png)

**Comment by DonJayamanne** [comment]
This requires sorry for ipywidgets, and yes this will be available in the next release.
Closing for now
